src/app/backend/apihandler.go

```diff
@@ -74,6 +74,15 @@ func CreateHttpApiHandler(client *client.Client) http.Handler {
 			Writes(Logs{}))
 	wsContainer.Add(logsWs)
 
+	eventsWs := new(restful.WebService)
+	eventsWs.Path("/api/events").
+		Produces(restful.MIME_JSON)
+	eventsWs.Route(
+		eventsWs.GET("/{namespace}").
```

Comment:
I see. That's fair enough explanation. We can merge as it is and then continue work. I'll let you decide. Do what it better for you.